src/functions-reference/multivariate_discrete_distributions.Rmd

```diff
@@ -68,48 +68,48 @@ scale.
 \frac{N!}{\prod_{k=1}^K y_k!}. \]
 
 ### Sampling statement
 
-`y ~ ` **`multinomial_logit`**`(theta)`
+`y ~ ` **`multinomial_logit`**`(gamma)`
 
-Increment target log probability density with `multinomial_logit_lupmf(y | theta)`.
+Increment target log probability density with `multinomial_logit_lupmf(y | gamma)`.
 `r since("2.24")`
 <!-- real; multinomial_logit ~; -->
 \index{{\tt \bfseries multinomial\_logit }!sampling statement|hyperpage}
 
 ### Stan functions
 
-<!-- real; multinomial_logit_lpmf; (array[] int y | vector theta); -->
-\index{{\tt \bfseries multinomial\_logit\_lpmf }!{\tt (array[] int y \textbar\ vector theta): real}|hyperpage}
+<!-- real; multinomial_logit_lpmf; (array[] int y | vector gamma); -->
+\index{{\tt \bfseries multinomial\_logit\_lpmf }!{\tt (array[] int y \textbar\ vector gamma): real}|hyperpage}
 
-`real` **`multinomial_logit_lpmf`**`(array[] int y | vector theta)`<br>\newline
+`real` **`multinomial_logit_lpmf`**`(array[] int y | vector gamma)`<br>\newline
 The log multinomial probability mass function with outcome array `y`
-of size $K$ given the $K$-simplex distribution parameter $\text{softmax}^{-1}(\theta)$ and
+of size $K$ given the log $K$-simplex distribution parameter $\gamma$ and
 (implicit) total count `N = sum(y)`
 `r since("2.24")`
 
-<!-- real; multinomial_logit_lupmf; (array[] int y | vector theta); -->
-\index{{\tt \bfseries multinomial\_logit\_lupmf }!{\tt (array[] int y \textbar\ vector theta): real}|hyperpage}
+<!-- real; multinomial_logit_lupmf; (array[] int y | vector gamma); -->
+\index{{\tt \bfseries multinomial\_logit\_lupmf }!{\tt (array[] int y \textbar\ vector gamma): real}|hyperpage}
 
-`real` **`multinomial_logit_lupmf`**`(array[] int y | vector theta)`<br>\newline
+`real` **`multinomial_logit_lupmf`**`(array[] int y | vector gamma)`<br>\newline
 The log multinomial probability mass function with outcome array `y`
-of size $K$ given the $K$-simplex distribution parameter $\text{softmax}^{-1}(\theta)$ and (implicit) total count `N = sum(y)` dropping constant additive
+of size $K$ given the log $K$-simplex distribution parameter $\gamma$ and (implicit) total count `N = sum(y)` dropping constant additive
 terms
 `r since("2.25")`
 
-<!-- array[] int; multinomial_logit_rng; (vector theta, int N); -->
-\index{{\tt \bfseries multinomial\_logit\_rng }!{\tt (vector theta, int N): array[] int}|hyperpage}
+<!-- array[] int; multinomial_logit_rng; (vector gamma, int N); -->
+\index{{\tt \bfseries multinomial\_logit\_rng }!{\tt (vector gamma, int N): array[] int}|hyperpage}
 
-`array[] int` **`multinomial_logit_rng`**`(vector theta, int N)`<br>\newline
-Generate a multinomial variate with simplex distribution parameter
-$\text{softmax}^{-1}(\theta)$ and total count $N$; may only be used in transformed data and
+`array[] int` **`multinomial_logit_rng`**`(vector gamma, int N)`<br>\newline
+Generate a multinomial variate with log simplex distribution parameter
```

Comment:
The first one would be right without the `^{-1}`.  The second one could be confusing because `softmax` isn't invertible.  How about the following.

```
Generate a variate from a multinomial distribution with probabilities `softmax(gamma)` and total count `N`.
```

This works around the problem of softmax not being invertible.  I do like changing the variate to `gamma` to not confuse it with the `theta` we tend to use for simplexes.

Also, it needs a final period (this was an earlier problem apparently).